SetupGuide_Plugin_Admin.php

```diff
@@ -779,8 +779,8 @@ public function get_lazyload_settings() {
 	 * @uses $_POST['enable']
 	 */
 	public function config_lazyload() {
-		if ( wp_verify_nonce( $_POST['_wpnonce'], 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
-			$enable           = ! empty( $_POST['enable'] );
+		if ( wp_verify_nonce( Util_Request::get_string( '_wpnonce' ), 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
```

Comment:
Remove the phpcs comment to have it validate the change.

Reply:
Removed 2 PHPCS disable comments.